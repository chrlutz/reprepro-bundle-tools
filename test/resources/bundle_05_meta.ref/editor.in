# Note: clean this file completely to CANCEL this current 'meta' action

= BUNDLE-METADATA =
Bundlename: mybionic/0001
BasedOn: NEW
Distribution: mybionic
Rollout: false
Target: plus
Creator: …USER…

= RELEASENOTES =
<Subject>

<Details>

__DYNAMIC_PACKAGE_LIST__

