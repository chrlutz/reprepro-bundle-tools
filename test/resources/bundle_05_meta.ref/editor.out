# Note: clean this file completely to CANCEL this current 'meta' action

= BUNDLE-METADATA =
Bundlename: mybionic/0001
BasedOn: NEW
Distribution: mybionic
Rollout: false
Target: plus
Creator: …USER…

= RELEASENOTES =
This is my best test bundle

<Details>

__DYNAMIC_PACKAGE_LIST__

